<launch>
    <arg name="id" default="1"/>

    <node pkg="atlas_drone" type="drone_terminal" name="drone_terminal_$(arg id)" output="screen">
        <param name="drone_id" value="$(arg id)" />
    </node>
</launch>
